docs/_includes/input-chips.html

```diff
@@ -0,0 +1,344 @@
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-2-2-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                            <span class="chip">
+                                <span id="chip-interactive-2-3-text" class="chip__text">Baseball</span>
+                                <button class="chip__button" type="button" aria-label="Remove" aria-describedby="chip-interactive-2-3-text">
+                                    <svg class="icon icon--close-12" focusable="false" width="13" height="12" aria-hidden="true">
+                                        {% include symbol.html name="close-12" %}
+                                    </svg>
+                                </button>
+                            </span>
+                        </div>
+                        <button type="button" class="icon-btn icon-btn--small input-chips_list-toggle" aria-label="Toggle options list" aria-haspopup="listbox" aria-owns="listbox-input-chips-2">
+                            <svg class="icon icon--chevron-down-16" focusable="false" height="16" width="16" aria-hidden="true">
+                                {% include symbol.html name="chevron-down-16" %}
+                            </svg>
+                        </button>    
+                    </div>
+                    <span class="combobox combobox--js input-chips__combobox">
+                        <span class="combobox__control input-chips_combobox__control">
+                            <input id="input-chips-2-input" role="combobox" type="text" placeholder="Add Sport" aria-haspopup="listbox" aria-owns="listbox-input-chips-2" />
+                        </span>
+                        <div class="combobox__listbox">
+                            <div id="listbox-input-chips-2" class="combobox__options" role="listbox">
+                                <div class="combobox__option" role="option">
+                                    <span>Baseball</span>
+                                </div>
+                                <div class="combobox__option" role="option">
+                                    <span>Hockey</span>
+                                </div>
+                                <div class="combobox__option" role="option">
+                                    <span>Tennis</span>
+                                </div>
+                            </div>
+                        </div>
+                    </span>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {% highlight html %}
+TBD
+    {% endhighlight %}
+
+    <h3>Input Chips with Helper Text</h3>
```

Comment:
Similar to the label comment above, we might want to consider removing any responsibility for helper text placement too, and have this module responsible only for the layout of elements that sit within the combobox boundary.